packages/react-dom/src/__tests__/ReactDOMFiber-test.js

```diff
@@ -748,153 +719,50 @@ describe('ReactDOMFiber', () => {
       }
     }
 
-    ReactDOM.render(<Parent />, container);
+    await act(async () => {
+      root.render(<Parent />);
+    });
     expect(container.innerHTML).toBe('');
     expect(portalContainer.innerHTML).toBe('<div>bar</div>');
   });
 
-  // @gate !disableLegacyContext
```

Comment:
Deleted this because they depend on legacy context, they still exist in `ReactDOMLegacyFiber`.